Sources/Controllers/2.0/Settings/OAImportCompleteViewController.mm

```diff
@@ -341,6 +355,13 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
         UIViewController* resourcesViewController = [[UIStoryboard storyboardWithName:@"Resources" bundle:nil] instantiateInitialViewController];
         [rootController.navigationController pushViewController:resourcesViewController animated:YES];
     }
+    else if (dataType == EOAImportDataTypeOsmNotes)
+    {
+        
+        UITabBarController* myPlacesViewController = [[UIStoryboard storyboardWithName:@"MyPlaces" bundle:nil] instantiateInitialViewController];
+        [myPlacesViewController setSelectedIndex:2];
```

Comment:
Check if OSM Editing plugin is enabled before opening the screen. If the plugin is disabled this will cause a crash.

Reply:
It's ok. If osm plugin is disabled then `[myPlacesViewController setSelectedIndex:2]` redirects to screen with index 0. There's no crash.

Reply:
Check added&